src/typechecker/FStar.TypeChecker.DeferredImplicits.fs

```diff
@@ -138,7 +138,7 @@ let solve_deferred_to_tactic_goals env g =
         in
         let goal_ty = U.mk_eq2 (env.universe_of env_lax t_eq) t_eq tp.lhs tp.rhs in
         let goal, ctx_uvar, _ =
-            Env.new_implicit_var_aux reason tp.lhs.pos env goal_ty Strict None
+            Env.new_implicit_var_aux reason tp.lhs.pos env goal_ty Allow_untyped None
```

Comment:
This is an important one. @aseemr @mtzguido The rationale is that this goal is being solved by a tactic and so doesn't need to be re-checked.